roles/seqreports/tasks/pre_reqs.yml

```diff
@@ -13,18 +21,24 @@
     path: "{{ seqreports_fastq_screen_resource_path }}"
   delegate_to: localhost
 
-- name: download fastq_screen genomes (please note this will be very slow)
-  command: python roles/seqreports/files/get_fastq_screen_reference_genomes.py {{ seqreports_fastq_screen_resource_path }}
+- name: download fastq_screen genomes (please note this will be very slow), with virtual_env
+  shell: "source {{ virtual_env }}/bin/activate && python roles/seqreports/files/get_fastq_screen_reference_genomes.py {{ seqreports_fastq_screen_resource_path }}"
+  args:
+    creates: "{{ seqreports_fastq_screen_resource_path }}/FastQ_Screen_Genomes"
+  delegate_to: localhost
+  when: has_virtual_env == true | bool
+
+- name: download fastq_screen genomes (please note this will be very slow), without virtual_env
+  shell: "python roles/seqreports/files/get_fastq_screen_reference_genomes.py {{ seqreports_fastq_screen_resource_path }}"
   args:
     creates: "{{ seqreports_fastq_screen_resource_path }}/FastQ_Screen_Genomes"
   delegate_to: localhost
-  when: false
+  when: has_virtual_env == false | bool
 
```

Comment:
as you noted above, the download location is not synced to the destination so the symlink will be broken on `miarka1` and `miarka2`. Thus, the files needs to be included in the sync but perhaps the larger goal could be to use indexes from the central, Uppmax-maintained reference data? On Irma, the fastq_screen config points to reference genome indexes like so:
```
## H. sapiens (Human)
DATABASE    Human   /sw/data/uppnex/igenomes/Homo_sapiens/Ensembl/GRCh37/Sequence/Bowtie2Index/genome BOWTIE2
```